MathlibTest/norm_num_ext.lean

```diff
@@ -118,15 +119,13 @@ example : Nat.minFac 121 = 11 := by norm_num1
 example : Nat.minFac 221 = 13 := by norm_num1
 example : Nat.minFac (2 ^ 19 - 1) = 2 ^ 19 - 1 := by norm_num1
 
-/-
-example : Nat.factors 0 = [] := by norm_num1
-example : Nat.factors 1 = [] := by norm_num1
-example : Nat.factors 2 = [2] := by norm_num1
-example : Nat.factors 3 = [3] := by norm_num1
-example : Nat.factors 4 = [2, 2] := by norm_num1
-example : Nat.factors 12 = [2, 2, 3] := by norm_num1
-example : Nat.factors 221 = [13, 17] := by norm_num1
--/
+example : Nat.primeFactorsList 0 = [] := by simp
+example : Nat.primeFactorsList 1 = [] := by simp
+example : Nat.primeFactorsList 2 = [2] := by simp
+example : Nat.primeFactorsList 3 = [3] := by simp
+example : Nat.primeFactorsList 4 = [2, 2] := by simp
+example : Nat.primeFactorsList 12 = [2, 2, 3] := by simp
+example : Nat.primeFactorsList 221 = [13, 17] := by simp
```

Comment:
Move these to a new test file, since it is not part of norm_num?

Reply:
Moved.